client/src/actions/auctionActionCreator.jsx

```diff
@@ -34,7 +34,7 @@ export function featuredArtsFetchedSuccess(featuredArts) {
   };
 }
 
-export function fetchingAnAuction(bool) {
+export function fetchingAnAuction() {
   return {
     type: 'FETCHING_AN_AUCTION',
     isFetchingAuction: true
```

Comment:
Seems that we are having a different version of code. In mine line 37 is: 
export function fetchingAuctions(bool){
  return {
    type: 'FETCHING_AUCTION',
    isFetching: bool
  };
}